libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -289,6 +284,10 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
             startDate: new Date(schedule.when.startDate), // check time zones
             isValidScheduleEmailData: true,
             repeat: parseRepeatString(schedule.when.recurrence),
+            unsubscribedRecipients:
```

Comment:
you don't need to take it from state - you can get it right from pros.editSchedule